src/osm_configurator/model/project/configuration/category.py

```diff
@@ -28,21 +31,27 @@ def __init__(self):
         Creates a new instance of a "Category" class.
         """
         self._active: bool = False
-        self._whitelist = []
-        self._blacklist = []
-        self._category_name = "Category Name"
-        self._calculation_method_of_area = calculation_method_of_area_enum_i.CalculationMethodOfArea.CALCULATE_SITE_AREA
-        self._attractivity_attributes = []
-        self._default_value_list = []
-        self._length = 3
+        self._whitelist: List = []
+        self._blacklist: List = []
```

Comment:
it would be good, if you were more accurate what type of list that is.
e.g. List[int] or List[str]